### ISSUE TITLE
Regression task

### ISSUE THREAD

**Issue by valer1435** [valer1435/RepoPilot#15]
I have dataset consist of table with different parameters of some fluid. Data in csv file, target column is 'fluid_stability'. I need fedot to solve regression task with my data. Please help, @RepoPilotAssistant

**Comment by RepoPilotAssistant** [comment]
To solve a regression task with your data using Fedot, follow these steps:

1. Import the necessary object:
   ```python
   from fedot import Fedot
   ```

2. Load and split your data into train and test sets. Fedot uses its own data object notation (InputData). You can create it from a file using the `InputData.from_dataframe()` method, providing a Task object with the type of task you want to solve:
   ```python
   from fedot.core.data.data import InputData
   from fedot.core.repository.tasks import Task, TaskTypesEnum

   data = InputData.from_dataframe(features_df, target_df, task=Task(TaskTypesEnum.regression))
   train, test = train_test_data_setup(data)
   ```

3. Initialize the FEDOT object and define the type of modeling problem as regression. You can also define parameters like seed for reproducibility and timeout in minutes:
   ```python
   model = Fedot(problem='regression', seed=42, timeout=5)
   ```

4. Train the model by calling the `fit()` method, which returns the best pipeline obtained during optimization:
   ```python
   best_pipeline = model.fit(train)
   ```

5. After fitting, you can view the structure of the resulting pipeline in text or plot format:
   ```python
   best_pipeline.print_structure()
   best_pipeline.show()
   ```

6. Obtain predictions for test data by calling the `predict()` method from the Fedot class:
   ```python
   prediction = model.predict(test)
   ```

7. Estimate the quality of predictions using the `get_metrics()` method:
   ```python
   print(model.get_metrics(rounding_order=4))
   ```

8. Optionally, save the model and load it later for inference:
   ```python
   best_pipeline.save(path='path_to_save_and_load')
   loaded_pipeline = Pipeline().load('path_to_save_and_load')
   ```

9. For inference on new data, create an InputData object with the new features and use the loaded pipeline to predict:
   ```python
   new_data_to_predict = InputData(features=new_features.values, target=None, idx=new_features.index.values, task=Task(TaskTypesEnum.regression), data_type=DataTypesEnum.table)
   prediction = loaded_pipeline.predict(new_data_to_predict).predict
   ```
